src/index.ts

```diff
@@ -247,7 +259,8 @@ export function Dictionary<V>() {
  */
 export function Record<O>(runtypes: {[K in keyof O]: Runtype<O[K]> }): Runtype<O> {
   return runtype(x => {
-    Dictionary().check(x)
+    if (x === null || x === undefined)
+      throw new ValidationError(`Expected a defined non-null value but was ${typeof x}`)
```

Comment:
It won't work. Dictionary do not accept strings, arrays, number, etc.
Record({}) does.